Backend/lib/shared-services/src/IssueManagement/Domain/Action/UI/Issue.hs

```diff
@@ -388,7 +388,7 @@ createIssueReport (personId, merchantId) mbLanguage Common.IssueReportReq {..} i
   config <- issueHandle.findMerchantConfig merchantId mocId (Just personId)
   processIssueReportTypeActions personId mbOption mbRide (Just config) True identifier issueHandle
   issueReport <- mkIssueReport mocId updatedChats shouldCreateTicket now
-  let isLOFeedback = (identifier == CUSTOMER) && (maybe False (any (T.isInfixOf $ T.toLower description) . map T.toLower) config.sensitiveWords)
+  let isLOFeedback = (identifier == CUSTOMER) && not (T.null description) && maybe False (any (\word -> T.toLower word `T.isInfixOf` T.toLower description)) (config.sensitiveWords)
```

Comment:
Either we can keep 'sensitiveWords' as Not Null in domain type and maybe in beam.
Or
If you're using it here only then pass sensitive words as empty list to checkSensitiveWords function so you don't have to write "maybe False" in both things